SUPPORTED-VERSIONS.md

```diff
@@ -5,13 +5,18 @@
 | **iOS**    |     ✅    |  `11+` |
 | **tvOS**   |     ✅    |  `11+` |
 | **iPadOS** |     ✅    |  `11+` |
-| **VisionOS** |   ⚠️    |  `1.1+` |
+| **visionOS** |   ⚠️    |  `1.0+` |
+| **macOS**  |     ⚠️    |  `10.15+` |
```

Comment:
What kind of support we are offering here, I see 
- Mac (I think this is most interesting)
- Mac Catalyst
- Mac Designed for iPad

<img width="380" alt="image" src="https://github.com/DataDog/dd-sdk-ios/assets/8882380/421ae0e5-8a1e-4d89-aa61-f6cb1e3ecebb">

Reply:
I made some adjustment based on previous comment. Now we only support 12+

Now target destination is a pandora box.

I'd like to discuss during the SDK meeting.
I think we can simplify and have only one target with all the destinations, instead of current iOS/tvOS split